README.md

```diff
@@ -25,3 +25,10 @@ feature branch, and [raise a pull request][raise-a-pr].
 
 [install-ruby]: https://www.ruby-lang.org/en/documentation/installation/
 [raise-a-pr]: https://github.com/srobo/docs/pull/new/master
+
+### Navigation Sidebar
+
+The docs navigation sidebar is generated from `_data/sidebar_tree.yaml` as part
+of the build process. This file is manually updated in order that we can
```

Comment:
```suggestion
of the build process. This file is manually updated in order so that we can
```

Reply:
I think the grammar here is already correct?

Reply:
Honestly the use of "in order" while discussing how the file is ordered is confusing.
```suggestion
of the build process. This file is manually updated so that we can
```